### PR TITLE
Add docker-entrypoint and README in docker path

### PR DESCRIPTION
**Pull request by gbecares**

### CODE REVIEW

File: docker/docker-entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/bin/bash -xe
+ tail -f /var/log/*
```
Comment (aargomaniz) [inline docker/docker-entrypoint.sh]: In /var/log path you will find a lot of different files so this tail will show kern.log, syslog, dmesg, auth.log and so on. In my opinion you can try to read only ingestion logs using "tail -f /var/log/sds/ingestion-*" or not tail any file.

Comment (gbecares) [inline docker/docker-entrypoint.sh]: It will be changed in the future due to script issues.

**Comment by gbecares** [comment]
:shipit:

**Comment by gbecares** [comment]
:shipit:

**Comment by stratiocommit** [comment]
Comment accepted. Will treat it ASAP

**Comment by stratiocommit** [comment]
Squashing and checks in progress.

**Pull request merged**